mix.exs

```diff
@@ -30,9 +30,10 @@ defmodule EctoFilter.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:ecto, "~> 2.1"},
-      {:postgrex, "~> 0.13", optional: true},
-      {:poison, "~> 3.1", optional: true},
+      {:ecto, "~> 3.0"},
+      {:ecto_sql, "~> 3.0"},
```

Comment:
Maybe we should broaden the ecto version requirements and make ecto 3 dependencies optional at least until we'll migrate all our repositories to ecto 3?

Reply:
for that reason hex versions exist, but as far as i understood this repo is not published to hex

Reply:
Yep. I'll publish it when its API became stable

Reply:
i don't think that support both versions of ecto is a good idea, since they have different queryable structure. Lets leave it as PR since we update all repos to ecto 3, separate branch is enough for now.

Reply:
Agree